apps/emails-and-messages/src/modules/app-configuration/ui/messaging-providers-box.tsx

```diff
@@ -80,9 +78,9 @@ export const MessagingProvidersBox = ({
   const getProviderLogo = (configuration: ConfigurationListItem) => {
     switch (configuration.provider) {
       case "smtp":
-        return <Image alt="SMTP logo" src={smtp} height={20} width={20} />;
+        return <SmtpLogo height={20} width={20} />;
       case "sendgrid":
-        return <Image alt="Sendgrid logo" src={sendgrid} height={20} width={20} />;
+        return <SendgridLogo height={20} width={20} />;
```

Comment:
I see that all the logos in this PR have a size of 20/20. Maybe we can generalize it?

```
type ProviderLogoProps = {
  providerName: "sendgrid" | "smtp";
  size: "small" | "medium";
}
const ProviderLogo = ({ providerName, size }: ProviderLogoProps) => ...
```

Reply:
Since I don't have use cases for big/small right now, I'll leave the current implementation. Don't want to introduce too many separate tokens apart of macaw ui.